src/cpp/src/generation_config.cpp

```diff
@@ -174,9 +174,6 @@ GenerationConfig greedy() {
     greedy_config.temperature = 0.0f;
     greedy_config.ignore_eos = true;
```

Comment:
I think it would be fair to remove ignore_eos, temperature and num_return_sequences as well